docs/installation/kubernetes.rst

```diff
@@ -235,4 +91,3 @@ initial installation:
 .. _Haven: https://haven.commonground.nl/
 .. _NGINX: https://www.nginx.com/
 .. _Ansible: https://www.ansible.com/
-.. _open_zaak_k8s: https://github.com/open-zaak/ansible-collection/tree/main/roles/open_zaak_k8s
```

Comment:
@SilviaAmAm do you know whether we will keep this `open-zaak/ansible-collection` repo or whether the content will be moved to https://github.com/maykinmedia/ansible-collection?

Reply:
For K8 deployments, all the ansible playbooks have been deleted. We now use the helm charts to deploy to K8!